build-support/bin/generate_json_schema_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
There are some additional smoke tests (for passing and failing configuration files) submitted via [schemastore ](https://github.com/SchemaStore/schemastore/pull/2723/files#diff-de00f525d704152a97d37822f083884c6c496d75c7779b0b3ef9639be4ad4418)